web-common/src/components/chip/removable-list-chip/RemovableListChip.svelte

```diff
@@ -93,6 +93,7 @@ are details left to the consumer of the component; this component should remain
   <RemovableListMenu
     {excludeStore}
     slot="floating-element"
+    let:toggleFloatingElement
```

Comment:
Why is this needed?

Reply:
See [the migration guide's section on default slot bindings](https://svelte.dev/docs/v4-migration-guide#default-slot-bindings).

Previously, we were using _default_ slot props (like `toggleFloatingElement`) in _named_ slots (like `slot="floating-element"`). But that's no longer allowed. Now, to migrate, I've added a _named_ slot prop `toggleFloatingElement`.